base/applications/rapps/appview.cpp

```diff
@@ -1424,86 +1266,77 @@ BOOL CAppsListView::SetViewMode(DWORD ViewMode)
     return SendMessage(LVM_SETVIEW, (WPARAM)ViewMode, 0) == 1;
 }
 
-BOOL CAppsListView::AddInstalledApplication(CInstalledApplicationInfo *InstAppInfo, LPVOID CallbackParam)
+BOOL CAppsListView::AddApplication(CApplicationInfo *AppInfo, BOOL InitialCheckState)
 {
-    if (ApplicationViewType != AppViewTypeInstalledApps)
+    if (ApplicationViewType == AppViewTypeInstalledApps)
     {
-        return FALSE;
-    }
+        /* Load icon from registry */
```

Comment:
you might consider switching to cpp style comments throughout this file (for the single lined comments at least).
which is what the majority of rapps implementation uses and which is shorter.